packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.base.tsx

```diff
@@ -106,8 +106,10 @@ export class MessageBarBase extends BaseComponent<IMessageBarProps, IMessageBarS
   }
 
   private _renderMultiLine(): React.ReactElement<React.HTMLAttributes<HTMLAreaElement>> {
+    const nativeProps = getNativeProps(this.props, divProperties);
+
     return (
-      <div className={this._classNames.root} aria-live={this._getAnnouncementPriority()}>
+      <div className={this._classNames.root} aria-live={this._getAnnouncementPriority()} {...nativeProps}>
```

Comment:
This won't conflict with https://github.com/OfficeDev/office-ui-fabric-react/pull/7854/files#diff-abc7a1995176f91564e37c7c3d3196efR144?

Reply:
I was just following where we have `aria-live={this._getAnnouncementPriority()}` and that's both here in `_renderMultiLine()` and in `_renderInnerText()`.

Reply:
Looking into it more though, it looks like we probably don't need `aria-live` and `role=status` on the span element in `_renderInnerText()`. 

According to the [W3C standards for aria-live](https://www.w3.org/TR/wai-aria-1.1/#aria-live)

> When the property is not set on an object that needs to send updates, the politeness level is the value of the nearest ancestor that sets the aria-live attribute.

The span in `_renderInnerText()` can just inherit the `aria-live` value from its ancestor, the root div. In that case, we only need to spread `nativeProps` and set `aria-live` on the root.

Reply:
However, according to the [W3C standards on status](https://www.w3.org/TR/wai-aria-1.1/#status):

> Elements with the role status have an implicit aria-live value of polite and an implicit aria-atomic value of true.

So, it someone were to pass in an empty value into `aria-live` through native props, the `role=status` region will still maintain its aria-live value of polite. Would we want to allow this to be overwritten by native props as well?

Reply:
@cschlechty and @micahgodbolt it looks like both of you have reviewed several MessageBar PRs. What do you think? Based on the expected functionality of MessageBar, would it make sense to only have the `aria-live` and `role=status` on the parent element, the root?